google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/CloudLibraryProjectState.java

```diff
@@ -47,7 +49,10 @@
 public class CloudLibraryProjectState implements ProjectComponent {
 
   private final Project project;
+
   private Map<Module, Set<CloudLibrary>> moduleLibraryMap = Maps.newHashMap();
+  private Map<Module, Optional<MavenDomDependency>> moduleBomMap = Maps.newHashMap();
```

Comment:
This would be a map of project module to a BOM XML element in pom.xml, correct? `MavenDomDependency` sound a bit generic, maybe worth adding a comment clarifying what exactly this maps contains?

Reply:
I named the variable `moduleBomMap` to try and make the clear. But I can definitely add a comment. Along the same lines, do you find that there is confusion with the `moduleLibraryMap` too?

Reply:
`moduleLibraryMap` seems pretty clear on the other hand, since it has a specifically named type (`CloudLibrary`) as a value.

Reply:
makes sense. ill update. thanks